### PR TITLE
Fix the example cmdlet of set oaa

### PR DESCRIPTION
**Pull request by LuChen-MSFT**
Fix the example of Set-Oaa cmdlet.

### CODE REVIEW

File: skype/skype-ps/skype/Set-CsOrganizationalAutoAttendant.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ The Set-CsOrganizationalAutoAttendant cmdlet lets you modify the properties of a
 ### -------------------------- Example 1 --------------------------
 ```
 $oaa = Get-CsOrganizationalAutoAttendant -PrimaryUri "sip:mainoaa@contoso.com"
-$oaa.LineUris = [System.Uri] "tel:+##########"
+$oaa.LineUris = @([System.Uri] "tel:+##########")
```
Comment (muwaqar) [inline skype/skype-ps/skype/Set-CsOrganizationalAutoAttendant.md]: Can you put a valid phone number here as well? like 1234567890

**Comment by muwaqar** [comment]
Posted one comment. Let me know when you have fixed it so that I can merge.

**Comment by doctordns** [comment]
Rather than a totally bogus number, at least make the number semi-real.
Like +14255551420 
rather than +1234567890

**Comment by LuChen-MSFT** [comment]
@doctordns Checked the document of New-CsOrganizationalAutoAttendant. I am going to use "tel:+11098765432", the number used in it, to keep consistency.

**Pull request merged**